### PR TITLE
[BISERVER-12297] created new endpoint for updating a schedule saving the original owner

### PR DESCRIPTION
**Pull request by andrey-markelov**
@pamval @pentaho-nbaker please review and merge

### CODE REVIEW

**Comment by andrey-markelov** [comment]
@pamval @pentaho-nbaker @dkincade please merge

**Comment by pamval** [comment]
@rmansoor can you please review this pull request ?

File: extensions/src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/JobScheduleRequest.java [hunk 1]
```diff
@@ -66,7 +67,7 @@
  */
 @XmlAccessorType( XmlAccessType.FIELD )
 @XmlType( name = "jobScheduleRequest", propOrder = { "actionClass", "complexJobTrigger", "cronJobTrigger", "duration",
-    "inputFile", "jobName", "jobParameters", "outputFile", "simpleJobTrigger", "timeZone" } )
+    "inputFile", "jobName", "jobParameters", "outputFile", "simpleJobTrigger", "timeZone", "jobId" } )
```
Comment (rmansoor) [inline extensions/src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/JobScheduleRequest.java]: A new attribute has been added "jobId". The documentation for the REST endpoint should reflect this change

**Comment by CodeOnCoffee** [comment]
This looks good to me.

**Comment by rmansoor** [comment]
@andrey-markelov looks good. Just upgrade the java doc header for JobScheduleRequest to include this new field JobId and we should be set. Thanks

**Comment by andrey-markelov** [comment]
updated

**Comment by rmansoor** [comment]
@andrey-markelov thanks

**Pull request merged**